editor.js

```diff
@@ -3,7 +3,7 @@
 	Extension Editor for XKit 7
 	Version 1.0
 
-	(c) 2011 - 2013 STUDIOXENIX
+	(c) 2011 - 2015 STUDIOXENIX & New-XKit Team
```

Comment:
> can't really be thought of as such from a copyright point of view

Are we not releasing the software as a group ("New XKit") under the GPLv3?  We don't have an official Contributor Agreement that our contributors sign before releasing their code to us, but at this point it's kind of implied, no?